code/workspaces/infrastructure-api/src/controllers/volumeController.js

```diff
@@ -90,6 +91,10 @@ const getByIdValidator = service.middleware.validator([
   check('id').exists().isMongoId().withMessage('id must be specified as a Mongo Id'),
 ], logger);
 
+const actionWithProjectKeyValidator = () => service.middleware.validator([
+  check('projectKey').exists().withMessage("Project 'projectKey' must be specified in URL."),
```

Comment:
Should the error just be `projectKey must be specified in the URL`?

Reply:
Correct, thanks.